src/ec/curve25519/x25519.rs

```diff
@@ -100,7 +100,7 @@ fn x25519_ecdh(
     let peer_public_key: &[u8; PUBLIC_KEY_LEN] = peer_public_key.as_slice_less_safe().try_into()?;
 
     #[cfg_attr(
-        not(all(not(target_os = "ios"), target_arch = "arm")),
+        not(all(not(target_os = "ios"), not(target_os = "tvos"), target_arch = "arm")),
```

Comment:
(came here from the [cc PR](https://github.com/rust-lang/cc-rs/pull/881) about tvOS as well)

There's [the x86_64 tvOS target](https://doc.rust-lang.org/nightly/rustc/platform-support/apple-tvos.html), although its just for the simulator running on a Intel Mac host. Rust doesn't support the original Apple TV, which is the only physical one to ever use an x86 chip.

> Note that macos is not in this list because there's no 32-bit ARM macOS (AFAIK).

this is correct though

Reply:
> tvos is aarch64-only, right? If so then you don't need the changes in this file, because this is for target_ach = "arm" (32-bit ARM).

Correct, I have overlooked it, will restore this file.